### DISCUSSION TITLE
One value on the chart

### DISCUSSION THREAD

**Discussion by popkovanton** [patrykandpatrick/vico#904] [Questions]
### Question

Could you please help me? How can I represent only one value on the chart in the middle?
But when I have 2 or more values, they should fill the chart

<img width="641" alt="Screenshot 2024-10-02 at 12 26 28" src="https://github.com/user-attachments/assets/bc1b5f9b-e8b8-427e-98ad-2b2899e36e70">

<img width="475" alt="Screenshot 2024-10-02 at 12 26 56" src="https://github.com/user-attachments/assets/f2fe1659-49bf-48d3-8f40-ab6e7122fe62">

<img width="472" alt="Screenshot 2024-10-02 at 12 27 53" src="https://github.com/user-attachments/assets/7d3b73bf-34e2-427f-a8aa-d4cc6fa0b022">

### Vico version(s)

2.0.0-alpha.22

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello, @popkovanton. We will follow up to this in #905.

**Reply by popkovanton** [reply]
The solution from https://github.com/patrykandpatrick/vico/discussions/905 didn't help me.
Could you explain to me how to center horizontally one value on Chart 1 sample? (2.0.0-beta.2)

**Reply by popkovanton** [reply]
I think I can do it this way. But maybe you have a better solution.

```
rangeProvider = CartesianLayerRangeProvider.fixed(
          minX = if (x.size == 1) 0.0 else null,
          maxX = if (x.size == 1) 2.0 else null,
        )
```

**Reply by Gowsky** [reply]
As I said under #905, it should be centered automatically, and that's what happens in our testing. We've confirmed it in `Chart1` for version `2.0.0-beta.2`. If you're seeing different behavior, please share a screenshot and the modifications that you made to the code.
